### ISSUE TITLE
[Desktop] Windows --disable-frame-rate-limit no longer allows Brave to exceed 60 FPS (was working fine 7 days ago)

### ISSUE THREAD

**Issue by regnaio** [brave/brave-browser#10705]
## Description 
According to Brave, my version is currently: Version 1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)

7 days ago, I was able to get ~300 FPS in Brave by setting Target in Brave -> Properties as:
"C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --disable-frame-rate-limit

Evidence that this was working 7 days ago with >100 FPS: https://forum.babylonjs.com/t/higher-framerate-causes-higher-jump-height/12252

Today, when I tried to load the same demo (https://www.babylonjs-playground.com/#3EDS3A#22), the FPS is stuck at 60 and cannot exceed 60.

## Steps to Reproduce
I did nothing. I think Brave self-installed an update that broke --disable-frame-rate-limit functionality

## Actual result:
As mentioned in Description, please see https://www.babylonjs-playground.com/#3EDS3A#22

## Expected result:
Expect --disable-frame-rate-limit to allow FPS to exceed 60 FPS. It was working with 300 FPS 7 days ago.

## Reproduces how often: 
Always now. Was working fine 7 days ago.

## Brave version (brave://version info)

I did not build anything from source. Just downloaded and installed Brave as a normal consumer.
Version 1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)

Brave	1.10.97 Chromium: 83.0.4103.116 (Official Build) (64-bit)
Revision	8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
OS	Windows 10 OS Version 1903 (Build 18362.900)
JavaScript	V8 8.3.110.13
Flash	(Disabled)
User Agent	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
Command Line	"C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --sync-url=https://no-thanks.invalid --enable-features=WebUIDarkMode,PasswordImport,AutoupgradeMixedContent,PassiveMixedContentWarning,MixedContentSiteSetting,DnsOverHttps --disable-features=AutofillServerCommunication,LookalikeUrlNavigationSuggestionsUI,SmsReceiver,TabHoverCards,TextFragmentAnchor,AutofillEnableAccountWalletStorage,AllowPopupsDuringPageUnload,NotificationTriggers,VideoPlaybackQuality --flag-switches-begin --flag-switches-end --enable-audio-service-sandbox --disable-sync
Executable Path	C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe
Profile Path	C:\Users\MY_USERNAME\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
Variations	5e3a236d-4113a79e

Note: Replaced my username with MY_USERNAME

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the dev channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by rebron** [comment]
cc: @iefremov 

@regnaio If you have Chrome, do you know if you're experiencing this same behavior? Also, we've had a couple releases since you filed. I imagine this is still reproducing in our latest 1.13.82?

**Comment by regnaio** [comment]
Just checked that it is reproing today on same machine

Version 1.18.70 Chromium: 87.0.4280.101 (Official Build) (64-bit)

**Comment by iefremov** [comment]
@regnaio It appears to me that this is purely an upstream problem - could you please check hot it works in Chrome/Chromium?

FWIW I've found a comment on crbug.com saying that someone was able to uncap the frame rate by enabling Vulkan https://bugs.chromium.org/p/chromium/issues/detail?id=1138080#c8

**Comment by regnaio** [comment]
This happens in Google Chrome/Chromium, Edge, Brave. None of these browsers can exceed 60 FPS

I just tried enabling Vulkan. It gave me a blank white screen on launching Chrome (could not interact at all with Chrome and disable the Vulkan flag to bring things back to normal). Took hours to figure out how to bypass this..

By adding --no-experiments to the Chrome target and then finally disabling the Vulkan flag

**Comment by regnaio** [comment]
>60 FPS can now be achieved on:
Version 1.18.77 Chromium: 87.0.4280.101 (Official Build) (64-bit)

**Comment by bsclifton** [comment]
Thanks for the additional info @regnaio 👍

**Comment by haxballer35** [comment]
I have the same problem. The fps code works in chrome but not in brave. The fps is fixed at 144. I was getting 500+ fps 3 days ago. I have to use brave because it is more fluent in the haxball brave browser. I have two operating systems, i use windows 7 for haxball and windows 7 The code doesn't work in brave, but it works in windows 10. I tried to do what is written. I disabled the vulkan flags, but again in 144, the code does not work. Why did this happen and what should I do? Pls help
